site/content/en/docs/administration/advanced/installation_automatic_annotation.md

```diff
@@ -94,6 +94,8 @@ description: 'Information about the installation of components needed for semi-a
   - The number of GPU deployed functions will be limited to your GPU memory.
   - See [deploy_gpu.sh](https://github.com/openvinotoolkit/cvat/blob/develop/serverless/deploy_gpu.sh)
     script for more examples.
+  - For some models (namely [SiamMask](/docs/manual/advanced/ai-tools#trackers) you will need an [Nvidia driver](https://www.nvidia.com/en-us/drivers/unix/)
+    version greater than or equal to 450.80.02.
```

Comment:
Why do you think the version should be more or equal 450.80.02?

Reply:
This information is taken from https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html

Reply:
Ok, I see